backend/svirt.pm

```diff
@@ -244,18 +174,17 @@ sub load_snapshot {
     my $rsp;
     my $post_load_snapshot_command = '';
     if (check_var('VIRSH_VMM_FAMILY', 'hyperv')) {
-        my $ps          = 'powershell -Command';
-        my %credentials = {hostname => get_required_var('VIRSH_GUEST'), password => get_var('VIRSH_GUEST_PASSWORD')};
```

Comment:
Previously (https://github.com/os-autoinst/os-autoinst/pull/1166/files#diff-447b3fde9d20bc30cc5c2ab06c1184deL166) `VIRSH_HOSTNAME`/`VIRSH_PASSWORD` was the default tuple for SSH connections, but on Hyper-V - due to a different architecture -, we need to connect to jump-host, hence the `VIRSH_GUEST`/`VIRSH_GUEST_PASSWORD` combination instead. Unless the defaults are no more, this should break on Hyper-V.

Reply:
Good catch. Thanks!
My fault was, that I only checked on VIRSH_VMM_FAMILY but didn't realized that we do have two different SSH credentials inside one family.
Can you give me some background why we sometimes need that jump-host and sometimes not?

Reply:
argh, github compare confused me and hide the relevant lines. 
Take a look to https://github.com/os-autoinst/os-autoinst/pull/1166/files#diff-447b3fde9d20bc30cc5c2ab06c1184deR209 which is called when a SSH-Connection gets created. It has the `VIRSH_GUEST/VIRSH_GUEST_PASSWORD` credentials for `VIRSH_VMM_FAMILY eq 'hyperv'`.

So I think we use  `VIRSH_GUEST/VIRSH_GUEST_PASSWORD` here.

Reply:
Ah, right, I didn't know this one got it.

The background info about Hyper-V is that openQA requires VNC but Hyper-V only provides RDP. Jump host is the place where we feed RDP data to Xvnc so openQA can consume it.